plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/HookManager.java

```diff
@@ -192,4 +192,18 @@ public void init() {
         });
     }
 
+    public void callConfigUpdated() {
+        pluginHooks.forEach((pluginName, completableHook) -> {
+            if (!completableHook.isCompleted()) {
+                return;
+            }
+
+            try {
+                completableHook.configUpdated();
+            }
+            catch (Throwable throwable) {
+                FunnyGuilds.getPluginLogger().error("Failed to configUpdate() for " + pluginName + " plugin hook", throwable);
```

Comment:
configUpdate() -> update config

Reply:
This would be semantically incorrect, as there is no config for a specific hook.